packages/calling-stateful-client/src/StreamUtils.ts

```diff
@@ -140,14 +154,20 @@ async function createViewVideo(
       internalContext.setRemoteRenderInfo(
         callId,
         participantKey,
-        streamId,
+        remoteStreamId,
```

Comment:
Now that we are tracking multiple streams for the local user to do we not get a stream id for the local user?

Reply:
We don't at all ☹️ we do have streamInfo.device.id but when switchSource is called that id cahnges are the underhood so becomes a really hard to maintain key to use